### DISCUSSION TITLE
How to hide a twitter announcement in the header?

### DISCUSSION THREAD

**Discussion by henrytien** [squidfunk/mkdocs-material#3852] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/7891852/165660078-04b8b048-4e95-41fd-99e7-d5cc2e76d895.png)
![image](https://user-images.githubusercontent.com/7891852/165660166-50c1fc88-9eef-4203-b190-494027de5844.png)

I have commented on this part, but Twitter is still on the website.

![image](https://user-images.githubusercontent.com/7891852/165661119-47801e24-2b93-4fe7-8633-2da2d430c1e0.png)

**Comment by squidfunk** [comment, marked as answer]
I'm afraid you need to give us some more information. Did you fork the theme? If so, did you go through the [build process](https://squidfunk.github.io/mkdocs-material/customization/#theme-development) when you made changes to overrides? If not, that's what you have to do. Forking should be a last resort, it's a much better approach to use [theme extension](https://squidfunk.github.io/mkdocs-material/customization/#extending-the-theme) which gives you almost the same possibilities.

**Reply by henrytien** [reply]
Ok, I got it, thank you so much, I just git clone this repo and run it.

**Comment by henrytien** [comment]
I still have a problem. It doesn't work, when I have changed it.
![image](https://user-images.githubusercontent.com/7891852/166097411-7c3a4625-bb4c-4b65-89d6-6e0141a7fc4f.png)

![image](https://user-images.githubusercontent.com/7891852/166097529-9e2c4633-ce1b-4701-8c27-ebac2d02c385.png)
It's like doesn't change.
![image](https://user-images.githubusercontent.com/7891852/166097552-11d62e0a-03db-4019-a3a0-a647d4c07a3d.png)

I also have built it in the console.
![image](https://user-images.githubusercontent.com/7891852/166097579-c127c583-0e02-47cd-8553-74d75d6e173d.png)

![image](https://user-images.githubusercontent.com/7891852/166097606-169abac7-f082-4d4d-8187-11c4db19e2d2.png)
Hope you can help me thank you very much!

**Reply by henrytien** [reply]
@squidfunk thanks~

**Reply by squidfunk** [reply]
Try `npm run build:all` to build the theme including overrides. Again, for your purposes, theme extension is a better idea.

<img width="717" alt="Bildschirmfoto 2022-04-30 um 10 20 12" src="https://user-images.githubusercontent.com/932156/166097982-cec17280-1f2f-4d14-9608-2628db689602.png">
